Maestro/subscriptions.json

```diff
@@ -781,6 +787,34 @@
         }
       }
     },
+    // Update dependencies in uwp-setup master
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/buildtools/master/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/master/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectn-tfs-ilc/master/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/standard/release/2.0.0/Latest.txt"
```

Comment:
Generally these should match what's in dependencies.props to avoid triggering when an update won't happen, and to reliably trigger when an update is actually needed: https://github.com/dotnet/uwp-setup/blob/04e76473a35fefa0422a7696faeb308bd394e30e/dependencies.props#L57-L76.

My understanding is you'll want to trigger on WCF and CoreFX. The `dependencies.props` is missing projectn-tfs-ilc at the moment.